CHANGELOG.md

```diff
@@ -1,7 +1,3 @@
 # Changelog
 
-## 2.0.0
```

Comment:
Sorry to be unclear, but adding a line under "Unreleased" heading in the CHANGELOG is super helpful.

(The release process which comes later, has to decide on a version number and then it can rename the "Unreleased" heading to the version number.)